tests/unit/test_hwIo.py

```diff
@@ -29,7 +29,10 @@ def test_apc(self):
 		"""
 		# Initially, our event isn't set
 		self.assertFalse(self.event.is_set())
-		# Queue a lambda as APC that sets the event
-		hwIo.bgThread.queueAsApc(lambda param: self.event.set())
+		# Queue a function as APC that sets the event
+
+		def apc(param: int):
+			return self.event.set()
```

Comment:
Should this be None?
Maybe this is a bit misleading

```suggestion
		def apc(param: int) -> None:
			self.event.set()
```